shared/dataflow/codeql/dataflow/internal/DataFlowImpl.qll

```diff
@@ -3391,6 +3324,8 @@ module MakeImpl<LocationSig Location, InputSig<Location> Lang> {
             private predicate localStepFromHidden(PathNodeImpl n1, PathNodeImpl n2) {
               n2 = localStep(n1) and
               n1.isHidden()
+              or
+              n2 = n1.getAnImplicitReadSuccessorAtSink(_)
```

Comment:
Is this a relevant addition? Isn't this just for identifying subpath-wrappers through hidden callables?

Reply:
It matters when you have a subpath that ends at a sink, for example https://github.com/github/codeql/pull/17300/files#diff-54bce82118abe557abb0786a2be4a5761bd9ba673e50323071469138c6d57a56R597.

Reply:
That anchored link doesn't work for me. But it makes sense 👍